.github/workflows/release.yml

```
@@ -0,0 +1,81 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Based on the tag v6 code below (without any condition), should this not be bound to `v6` tags?
```suggestion
      - 'v6*'
```

Otherwise any tag would update the `6` mutable tag.

---

Good point.  Currently scanner image's immutable tag, say `3.567`, doesn't have the prefix of the mutable tag, say `5` or `6`, so this prefix won't work.  It's a good idea to sync them though.  We will revisit later to see if we can make them consistent.

---

Upon further pondering, I think the `Tag v6 scanner` seems to be better off as a separate workflow.
That way when cutting release candidate tags, it won't mistakenly update the v6 tag. Alternatively, we could just add an `if` on the specific step to ensure only "production like" tags causes an update on `6`.

---

That's a great idea.  I've implemented it in the latest round.